src/i965_encoder.c

```diff
@@ -1460,6 +1467,11 @@ intel_enc_hw_context_init(VADriverContextP ctx,
             if (encoder_context->codec == CODEC_H264)
                 encoder_context->context_roi = 1;
         }
+        if (obj_config->attrib_list[i].type == VAConfigAttribEncMaxSlices) {
```

Comment:
VAConfigAttribEncMaxSlices is a read-only attribute, user needn't provide VAConfigAttribEncMaxSlices when creating a VA context. so it is possible that the attrib list doesn't have VAConfigAttribEncMaxSlices.